client/signup/steps/clone-jetpack/index.jsx

```diff
@@ -0,0 +1,104 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import SignupActions from 'lib/signup/actions';
+import TileGrid from 'components/tile-grid';
+import Tile from 'components/tile-grid/tile';
+
+class CloneJetpackStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		goToNextStep: PropTypes.func.isRequired,
+		positionInFlow: PropTypes.number,
+		signupProgress: PropTypes.array,
+		stepName: PropTypes.string,
+		signupDependencies: PropTypes.object,
+	};
+
+	selectNew = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'new',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	selectMigrate = () => {
+		SignupActions.submitSignupStep( { stepName: this.props.stepName }, [], {
+			cloneJetpack: 'migrate',
+		} );
+
+		this.props.goToNextStep();
+	};
+
+	renderStepContent = () => {
+		const { originSiteName, destinationSiteName, translate } = this.props;
+
+		return (
+			<TileGrid>
+				<Tile
+					className="clone-jetpack__keep"
+					buttonLabel={ 'Keep plan where it is' }
+					description={ translate( 'Your plan would remain on %(originSiteName)s.', {
```

Comment:
I think this should be `will` instead of `would` because if user selects this option then it will stay there for sure.